src/Core/CommandBus/ExecutedCommandRegistry.php

```diff
@@ -115,7 +115,7 @@ private function getTrace(): array
         $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, self::BACKTRACE_LIMIT);
 
         foreach ($trace as $step) {
-            if ($step['class'] === TacticianCommandBusAdapter::class
+            if ($step['class'] === MessengerCommandBusAdapter::class
```

Comment:
It's not due to your PR as the problem was here before but I just want to mention that the MessengerCommandBusAdapter should be under Adapter namespace and then not using here (in the Core namespace)

Reply:
I agree, I'm think maybe this could be replaced with:

```suggestion
            if ($step['class'] instanceof CommandBusInterface::class
```

Reply:
And the new `MessengerCommandBusAdapter` could indeed be moved into the Adapter namespace, or actually maybe more in PrestaShopBundle since it's a bridge between the core interface and the Symfony messenger component